src/network/networkd-link.c

```diff
@@ -623,6 +623,12 @@ int link_stop_clients(Link *link, bool may_keep_dhcp) {
                         r = log_link_warning_errno(link, k, "Could not stop DHCPv4 client: %m");
         }
 
+        if (link->dhcp_acd) {
+                k = sd_ipv4acd_stop(link->dhcp_acd);
```

Comment:
Hmm, maybe we should make `sd_ipv4acd_stop()` do nothing on NULL?

Reply:
Year, I will create another PR later.